SU2_CFD/include/output/filewriter/CCGNSFileWriter.hpp

```diff
@@ -0,0 +1,143 @@
+/*!
+ * \file CCGNSFileWriter.hpp
+ * \brief Headers fo paraview binary file writer class.
+ * \author G. Baldan
+ * \version 7.2.0 "Blackbird"
+ *
+ * SU2 Project Website: https://su2code.github.io
+ *
+ * The SU2 Project is maintained by the SU2 Foundation
+ * (http://su2foundation.org)
+ *
+ * Copyright 2012-2021, SU2 Contributors (cf. AUTHORS.md)
+ *
+ * SU2 is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * SU2 is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with SU2. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#ifdef HAVE_CGNS
+#include "cgnslib.h"
+#endif
+
+/*--- Define data precision of output (float or double). ---*/
+typedef float dataPrecision;
+
```

Comment:
```suggestion
```

Reply:
This should be private for the class and not visible when the header is included.